admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.js

```diff
@@ -258,47 +259,58 @@ export default class CombinationsManager {
         }
       );
     });
+
+    this.eventEmitter.on(CombinationEvents.listRendered, () => {
+      // Init DeltaQuantityInput when each container of deltaQuantity is already rendered in DOM.
+      new DeltaQuantityInput({
+        submittableInputConfig: {
+          wrapperSelector: CombinationsMap.quantityInputWrapper,
```

Comment:
```suggestion
          wrapperSelector: CombinationsMap.deltaQuantityInputWrapper,
```

This should be renamed to clearly understand it's the delta field which is used